src/styles/blocks/modifiers.scss

```diff
@@ -0,0 +1,18 @@
+@for $day-number from 1 through 31 {
+  .calendar__day:nth-child(#{$day-number})::before {
+    content: '#{$day-number}';
+    color: #000;
+    padding-left: 1px;
+  }
+
+  .calendar--month-length-#{$day-number}
+    .calendar__day:nth-child(-n + #{$day-number}) {
+    display: flex;
+  }
```

Comment:
The `display: flex` property seems out of place here. The calendar cells should be displayed as block elements, and the positioning of the cells should be handled by the parent container (.calendar). Consider removing this property.